_docs/changelog.md

```diff
@@ -7,7 +7,15 @@ permalink: /docs/changelog/
 {:toc}
 
 Current version of the API is `1`
-Current version of the SDKs are `1.13.0`
+Current version of the SDKs are `1.14.1`
+
+1.14.0
+-----
+* Update `ZAR::Bank` business payout details
+
+1.14.0
+-----
+* Add support for the `ZAR::Bank` business payouts
```

Comment:
Shouldn't we mention the changes for the patch? (Like in the 1.12.2 below)

Reply:
Also, shouldn't we be having only 1 description of changes per release? here we have 1.14.0 twice...

Reply:
yep, it was a typo changed it to 1.14.1

Reply:
One more nitpick... should we be more explicit in the changes description? Like for instance mentioning the `legal_entity_type` change? As for https://github.com/transferzero/api-documentation/blob/c36ff8990f7d820c1f3c9dc9bba0f83011aecc8e/_docs/changelog.md#141...

Reply:
change made